lib/flop_phoenix/form_data.ex

```diff
@@ -0,0 +1,143 @@
+defimpl Phoenix.HTML.FormData, for: Flop.Meta do
+  import Flop.Phoenix.Misc, only: [maybe_put: 3]
+
+  def to_form(meta, opts) do
+    {name, opts} = name_and_opts(meta, opts)
+    {errors, opts} = Keyword.pop(opts, :errors, [])
+    {hidden, opts} = Keyword.pop(opts, :hidden, [])
+    {params, opts} = Keyword.pop(opts, :params, %{})
+    id = Keyword.get(opts, :id) || name || "flop"
+
+    %Phoenix.HTML.Form{
+      data: meta.flop,
+      errors: errors,
+      hidden: hidden_inputs(meta.flop, hidden),
+      id: id,
+      impl: __MODULE__,
+      name: name,
+      options: opts,
+      params: params,
+      source: meta
+    }
+  end
+
+  def to_form(
+        meta,
+        %{data: %Flop{} = flop} = form,
+        :filters,
+        opts
+      ) do
+    no_unsupported_options!(opts)
+
+    {id, opts} = Keyword.pop(opts, :id)
+    {default, opts} = Keyword.pop(opts, :default, [])
+    {prepend, opts} = Keyword.pop(opts, :prepend, [])
+    {append, opts} = Keyword.pop(opts, :append, [])
+    {skip_hidden_op, opts} = Keyword.pop(opts, :skip_hidden_op, false)
+
+    name = if form.name, do: form.name <> "[filters]", else: "filters"
+    id = if id = id || form.id, do: to_string(id <> "_filters"), else: "filters"
+    filters = if flop.filters == [], do: default, else: flop.filters
+    filters = prepend ++ filters ++ append
+
+    for {filter, index} <- Enum.with_index(filters) do
+      index_string = Integer.to_string(index)
+
+      hidden =
+        if skip_hidden_op,
```

Comment:
what is this good for?

Reply:
If this option is not passed (or is `false`), hidden inputs for both the `:op` and the `:field` are rendered. If it is `true`, only the hidden input for the `:field` is rendered. This doesn't make a lot of sense as it is, since you may want to configure this per field, so I'll probably move this to the field configuration opts passed to `inputs_for`.